tests/meta/report/test_mip_dna_api.py

```diff
@@ -1,14 +1,14 @@
 from cg.constants import REPORT_GENDER
+from tests.store_helpers import StoreHelpers
 
 
 def test_get_sample_metadata(
-    report_api_mip_dna, mip_analysis_api, case_mip_dna, sample_store, helpers
+    report_api_mip_dna, mip_analysis_api, case_mip_dna, sample_store, helpers: StoreHelpers
 ):
     """Tests sample metadata extraction."""
 
     # GIVEN a mock sample and the latest metadata
-    sample = helpers.add_sample(sample_store)
-    sample.internal_id = "ADM1"
```

Comment:
With version 1.4, this update of the internal id causes a unique constraint error:
```
E       sqlalchemy.exc.IntegrityError: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely)
E       (sqlite3.IntegrityError) UNIQUE constraint failed: sample.internal_id
E       [SQL: UPDATE sample SET internal_id=?, reads=? WHERE sample.id = ?]
E       [parameters: ('ADM1', None, 13)]
E       (Background on this error at: http://sqlalche.me/e/14/gkpj)
```
I'm not sure why it does not cause an error with version 1.3, but it seems like the auto flush behaviour differs somehow. Or maybe it is related to the new auto begin behaviour.